pages/dkp/konvoy/2.2/choose-infrastructure/aws/quick-start-aws/index.md

```diff
@@ -48,11 +48,10 @@ Before starting the Konvoy installation, verify that you have:
 
 ## Create a new AWS Kubernetes cluster
 
-If you use these instructions to create a cluster on AWS using the DKP default settings without any edits to configuration files or additional flags, your cluster is deployed on an [Ubuntu 20.04 operating system image][supported-systems] with 3 control plane nodes, and 4 worker nodes.
+If you use these instructions to create a cluster on AWS using the DKP default settings without any edits to configuration files or additional flags, your cluster is deployed on an [Ubuntu 20.04 operating system image][supported-systems] with 3 control plane nodes, and 4 worker nodes.  Essentially, a default AWS image is created which is not recommended for production use.
```

Comment:
```suggestion
If you use these instructions to create a cluster on AWS using the DKP default settings without any edits to configuration files or additional flags, your cluster is deployed on an [Ubuntu 20.04 operating system image][supported-systems] with 3 control plane nodes, and 4 worker nodes.
```

An image is not created. The image is used.

Reply:
Done!  Added your comments to all the sections and fixed the links inside the NOTE to point to Azure for the sections about Azure rather than AWS.